docs/webhooks/index.html

```diff
@@ -328,7 +449,127 @@ <h3 id="companyTestSessionFinished">companyTestSessionFinished</h3>
+    candidateEmail: 'john.doe@gmail.com',
+    candidateName: 'John Doe',
+    rejectedReasons: ['Two humans in the room', 'Screen not visible]
+    }
+};</code></pre>
+                    </div>
+
+                <div class="subsection">
+                    <h4 id="preScreenResultVerified">preScreenResultVerified</h4>
+                    <p>
+                        Fired when a CodeSignal auditor verifies a proctored test session.
+                    </p>
+                    <div>Request format:</div>
+                    <pre><code>{
+  event: 'preScreenResultVerified',
+  triggeredOn: number,
+  payload: {
+    testSessionId: string,
+    testId: string,
+    testTitle: string,
+    candidateEmail: string,
+    candidateName: string,
+    duration: number, // milliseconds
+    score: number,
+    maxScore: number,
+    codingScore?: number, // Coding Score in range [300, 850], if configured for this assessment
+    plagiarismLevel: number,
+    plagiarismLabel: 'none' | 'low' | 'medium' | 'high' | '-',
+    url: string,
+  }
+};</code></pre>
+                    <div>Sample request:</div>
+                    <pre><code>{
+  event: 'preScreenResultVerified',
+  triggeredOn: 1553720789347,
+  payload: payload: {
+    testSessionId: 'lehu382hdleh29',
+    testId: 'dhey29hcl28hl28',
+    testTitle: 'Software Engineer intern',
+    candidateEmail: 'john.doe@gmail.com',
+    candidateName: 'John Doe',
+    duration: 3480000, // 58 minutes
+    score: 900,
+    maxScore: 1000,
+    codingScore: 820,
+    plagiarismLevel: 0.5,
+    plagiarismLabel: 'medium',
+    url: https://app.codesignal.com/test-result/oH3qeBC38oFsf7qB4?accessToken=A7HnBpab4aD7xm7Kp-iPhk2se5Wnxeg7x9GruANLzn
+  }
+};</code></pre>
+                </div>
+
+
+<div class="subsection">
+    <h3 id="clientworkflow">Company Client workflow</h3><br>
+    <img alt="Diagram of webhooks triggered on client flow" width="100%" src="../images/client-events.png" />
+</div>
+                <div class="subsection">
+                    <h3 id="companyTestSessionCreated">companyTestSessionCreated</h3>
+                    <p>
+                        Fired when a test session is created.
+                    </p>
+                    <div>Request format:</div>
+                    <pre><code>{
+   event: 'companyTestSessionCreated',
+   triggeredOn: number,
+   payload: {
+     testSessionId: string,
+     testId: string,
+     testTitle: string,
+     candidateEmail: string,
+     candidateName: string
+   }
+ };</code></pre>
+                    <div>Sample request:</div>
+                    <pre><code>{
+   event: 'companyTestSessionCreated',
+   triggeredOn: 1553720789347,
+   payload: {
+     testSessionId: 'lehu382hdleh29',
+     testId: 'dhey29hcl28hl28',
+     testTitle: 'Software Engineer intern',
+     candidateEmail: 'john.doe@gmail.com',
+     candidateName: 'John Doe'
+   }
+ };</code></pre>
+</div>
+
+
+		<h2>Events for CodeSignal Certify</h2>
```

Comment:
Maybe out-of-scope, but might be nice to update "CodeSignal Test" to "CodeSignal Pre-Screen" and "CodeSignal Certify" to "CodeSignal Pre-Screen Certify" where it's possible to do so within this document?